openvpn/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for openvpn.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+updateStrategy: {}
+  # type: RollingUpdate
+  # rollingUpdate:
+  #   maxSurge: 1
+  #   maxUnavailable: 0
+
+image:
+  repository: kylemanna/openvpn
+  tag: 2.4
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  # type: NodePort
+  type: LoadBalancer
+  externalPort: 443
+  internalPort: 443
+  # hostPort: 443
+  externalIPs: []
+  # nodePort: 32085
+  # clusterIP: None
+  # LoadBalancerSourceRanges: 0.0.0.0/0
+  # loadBalancerIP: 10.0.0.1
+  annotations: {}
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: {}
+
+affinity: {}
+
+podAnnotations: {}
+
+readinessProbe:
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  successThreshold: 2
+
+persistence:
+  # storageClass: gp2-encrypted
```

Comment:
why is this commented out?

Reply:
Because it's optional, I kept the defaults generic

Reply:
@iuriaranda I've updated the chart a bit + clarified (it wouldn't have worked without defining the value actually.